CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Explicitly allow http connections via a new 'http' option to `--insecure-options` ([#1945](https://github.com/coreos/rkt/pull/1945)). Any data and credentials will be sent in the clear.
 - When using `bash`, `rkt` commands can be auto-completed ([#1955](https://github.com/coreos/rkt/pull/1955)).
 - The executables given on the command line via the `--exec` parameters don't need to be absolute paths anymore ([#1953](https://github.com/coreos/rkt/pull/1953)). This change reflects an update in the appc spec since [v0.7.2](https://github.com/appc/spec/releases/tag/v0.7.2). See rkt's [rkt run --exec](https://github.com/coreos/rkt/blob/master/Documentation/subcommands/run.md#overriding-executable-to-launch) documentation.
+- Redone the image stage1 handling. Replaced --stage1-image flag with --stage1-url, --stage1-path, --stage1-name and --stage1-hash flags and added a new --stage1-from-dir flag which loads stage1 image from the file in the default stage1 images directory. The directory can be configured at build-time and via the configuration file. ([#1977](https://github.com/coreos/rkt/pull/1977))
```

Comment:
It should be rephrased to say what this change provides from a user perspective. Faster with `--stage1-name=` or `--stage1-hash=` because it would not need to untar the stage1 image? What else?

Since we are getting closer to the version 1.0, we should start having some new sections in the release notes:

```
#### Migration

- The run-time flag `--stage1-image=` does not exist anymore. Scripts using this should be updated to use ...

#### Note for packagers

- New build-time configure flag `--with-stage1-default-images-directory`. Distributions are advised to set it to...
```